src/components/Comments/WriteComment.js

```diff
@@ -87,10 +95,11 @@ const WriteComment = ({
           <ToggleSwitch
             value={isAnonimity}
             onValueChange={toggleSwitch}
-            labelLeft="Anonymity"
+            labelLeft={withAnonymityLabel ? 'Anonymity' : null}
             backgroundActive={COLORS.lightgrey}
             backgroundInactive={COLORS.lightgrey}
             styleLabelLeft={styles.switch}
+            isViewOnly={isViewOnly}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `ToggleSwitch` component now receives an additional prop `isViewOnly` and the `labelLeft` prop is now conditionally set based on the `withAnonymityLabel` prop. Make sure that these changes do not affect the functionality of the `ToggleSwitch` component.



<!-- This is an auto-generated comment by OSS CodeRabbit -->